EIPS/eip-1.md

```diff
@@ -266,6 +286,79 @@ Many EIPs are written and maintained by developers with write access to the Ethe
 
 The editors don't pass judgment on EIPs. We merely do the administrative & editorial part.
 
+## Hardfork Meta
+
+Any changes to clients that effect consensus of the Ethereum Network results in a hardfork (See the `CORE` category of EIPs). There is then a "fork" in the blockchain where one chain follows the updated ruleset. A contentious fork is when a network split occurs because two communities form to continue supporting both the old and the new client codebases simultaneously. A split is not the typical case, as historically most upgrades have been adopted by the network wholly.
+
+Non-contentious forks, or a Network Upgrades, are part of the standard upgrade process. Core EIPs follow the EIP flow defined previously in EIP-1, are implemented into the client code with a block activation number, and released as a software upgrade to the clients. Node operators upgrade their clients, and at the specified block height, the network upgrade is deployed.
+
+The following is the specification and definition of the Hard Fork process for Ethereum, governed by the AllCoreDevs Community Calls.
```

Comment:
IIRC there was some controversy on EthMagcians about whether the process is governed by AllCoreDevs call *or* by the EIPs repo.

Reply:
@fulldecent I think you were the one suggesting it was governed by the repo.

Reply:
For CoreEIPs the process is governed by the ACD Call. To get into EFI an EIP needs to be brought up on the ACDs call. To get into a Hardfork you need to first get into EFI. Thus, anything that goes through EFI is explicitly governed by the ACDs Call.